MapboxAndroidDemo/src/main/java/com/mapbox/mapboxandroiddemo/model/usermodel/UserResponse.java

```diff
@@ -0,0 +1,188 @@
+package com.mapbox.mapboxandroiddemo.model.usermodel;
+
+import com.google.gson.annotations.SerializedName;
+
+import org.json.JSONObject;
+
+/**
+ * Created by LangstonSmith on 5/9/17.
+ */
+
+public class UserResponse {
+
+  @SerializedName("authorizations")
```

Comment:
Why `@SerializedName`? It's normally used for customizing field names (here the names are equal).
Further information on [`@SerializedName`](https://google.github.io/gson/apidocs/com/google/gson/annotations/SerializedName.html).